src/agentscope/agents/agent.py

```diff
@@ -48,7 +49,8 @@ def __init__(
             sys_prompt (`Optional[str]`):
                 The system prompt of the agent, which can be passed by args
                 or hard-coded in the agent.
-            model_config_name (`str`, defaults to None):
+            config_name_or_model (`[str, Callable[..., Any]`, defaults to
+            None):
                 The name of the model config, which is used to load model from
```

Comment:
"name of the model config to load model from configuration, or a callable instance of model for inference"? Same for DialogAgent and other agents